Automattic/wp-calypso
# Payment Methods: Add toggle for backup payment methods

#### Changes proposed in this Pull Request

This is part of a project to allow marking payment methods as available for "backup" which means that the payment method may be used if another payment method fails during an auto-renew. This PR adds a toggle to each payment method in the list of payment methods to allow toggling that method's "backup" status.

Depends on D67557-code which adds the appropriate endpoints.

#### Screenshots

<img width="1020" alt="Screen Shot 2021-09-30 at 6 56 21 PM" src="https://user-images.githubusercontent.com/2036909/135544238-9d3a7301-ecbc-4414-8150-64ca2bdceacc.png">

<img width="432" alt="Screen Shot 2021-09-30 at 7 11 42 PM" src="https://user-images.githubusercontent.com/2036909/135544243-f5999627-63b0-417d-ae4a-2ae45cb9d9db.png">

#### Testing instructions

- Visit `/me/purchases/payment-methods` (or the similar site-level page; they both use the same components).
- Verify that you see some indication that each payment method is not marked as a backup.
- Click to change a payment method to backup.
- Verify that the payment method changes to indicate that it's now a backup.
- Reload the page and verify that the payment method remains a backup.
